browser-extension/addModal.css

```
@@ -0,0 +1,24 @@
+#x-tournesol-modal {
+    top: 0;
+    left: 0;
+    position: fixed;
+
+    width: 100%;
+    height: 100%;
+
+    z-index: 1001;
```

The YouTube top bar (`#masthead-container`) uses `z-index: 2020`, so we'll need to go higher :)
```suggestion
    z-index: 9999;
```